package.json

```diff
@@ -11,16 +11,16 @@
     "src"
   ],
   "dependencies": {
-    "q": "~1.4.1",
-    "request": "~2.67.0",
-    "tar": "~2.1.1",
-    "tmp": "~0.0.28"
+    "q": "~1.5.0",
+    "request": "~2.81.0",
+    "tar": "~2.2.1",
+    "tmp": "~0.0.31"
   },
   "devDependencies": {
-    "mocha": "~2.3.4",
-    "mocha-junit-reporter": "~1.9.1",
-    "nock": "~5.2.1",
-    "sinon": "~1.17.2"
+    "mocha": "~3.2.0",
+    "mocha-junit-reporter": "~1.13.0",
+    "nock": "~9.0.9",
```

Comment:
This version is not supported on node 5.x (which isn't supported, so probably fine). According to nock that version supports up to nock 8.x